src/query/service/src/sql/planner/plans/create_tabular_function.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright 2022 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+use std::sync::Arc;
+
+use common_datavalues::DataSchema;
+use common_datavalues::DataSchemaRef;
+
+#[derive(Clone, Debug)]
+pub struct CreateTabularFunctionPlan {
```

Comment:
@BohuTANG @sundy-li  Need help. I am not sure how to handle this plan in the interpreter. I use `DataSchemaRef` for args and source so that it can't be used in meta crate like stage or udf. This plan seems more like view or table which has different interpreter logic with stage/udf.

Reply:
Need a new TabularFunctionInterpreter?